specification/logs/events/semantic_conventions/browser.md

```diff
@@ -0,0 +1,205 @@
+
+**[1]:**  Alias for [`http.url`](../../../trace/semantic_conventions/http.md)
+**[2]:** The URL fragment may be included for virtual pages
+
+`type` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `0` | physical_page - Initial page load within the browser and will generally also precede a PageNavigationTiming event.|
+| `1` | virtual_page - This is for Single Page Applications (SPA) where the framework provides the ability to perform client side only page "navigation", the exact definition of what a virtual page change is determined by the SPA and the framework it is using.|
+
+<details>
+<summary>Sample PageView Event</summary>
+
+```json
+    "log_record": {
+        "timeUnixNano":"1581452773000000789",
+        "attributes": [
+        {
+            "key": "event.domain",
+            "value": {
+            "stringValue": "browser"
+            }
+        },
+        {
+            "key": "event.name",
+            "value": {
+            "stringValue": "page_view"
+            }
+        },
+
+        {
+            "key": "event.data",
+            "value": {
+                "kvlistValue": [
+                    {
+                        "key": "type",
+                        "value": {
+                        "intValue": "0"
+                        }
+                    },
+                    {
+                        "key": "url",
+                        "value": {
+                        "stringValue": "https://www.guidgenerator.com/online-guid-generator.aspx"
+                        }
+                    },
+                    {
+                        "key": "referrer",
+                        "value": {
+                        "stringValue": "https://wwww.google.com"
+                        }
+                    },
+                    {
+                        "key": "title",
+                        "value": {
+                        "stringValue": "Free Online GUID Generator"
+                        }
+                    },
+                ]
+            }
+        }
+        ]
+    }
+```
+
+</details>
+
+### PageNavigationTiming
+
+**Event name**:`page_navigation_timing`
+
+This event describes the timing metrics of a page navigation as provided by
+`PerformanceNavigationTiming` Performance API.
+
+| Key  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| fetchStart | long | || Recommended |
+| unloadEventStart | long | || Recommended |
+| unloadEventEnd | long | || Recommended |
+| domInteractive | long | || Recommended |
+| domContentLoadedEventStart | long | || Recommended |
+| domContentLoadedEventEnd | long | || Recommended |
+| domComplete | long | || Recommended |
+| loadEventStart | long | || Recommended |
+| loadEventEnd | long | || Recommended |
+| firstPaint | long | || Recommended |
+| firstContentfulPaint | long | || Recommended |
+
+### ResourceTiming
+
+**Event name**:`resource_timing`
```

Comment:
Shouldn't this include an attribute denoting the resource that was fetched?

Reply:
Added `name` property that denotes the URL of the resource fetched. This is as per https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/name